src/containers/ResourceEmbed/components/ResourceEmbed.tsx

```diff
@@ -205,13 +229,30 @@ const ResourceEmbed = ({ id, type, isOembed }: Props) => {
           <PageContent variant="article" gutters="tabletUp">
             <PageContent variant="content" asChild>
               <ArticleWrapper>
-                <ArticleTitle title={properties.title} id={SKIP_TO_CONTENT_ID} contentType={type} />
+                <ArticleTitle
+                  title={properties.title}
+                  id={SKIP_TO_CONTENT_ID}
+                  contentType={type}
+                  heartButton={
+                    !hasLicensedContent(data?.resourceEmbed?.meta!, checkIfCopyrighted) && (
```

Comment:
Blir denne for streng? Hva med lyder som har et copyright-bilde?

Reply:
Fulgte bare logikken som den ble brukt dypere, litt usikker hvordan ellers vi skal løse det?

Reply:
Aiit, det kan vi da fikse 👍🏿